### ISSUE TITLE
Knowledge articles display without stying on iOS 11.3

### ISSUE THREAD

**Issue by chrisperkins** [forcedotcom/ServiceSDK-iOS#17]
SDK 212.0.2
iOS 11.3
iPhone & iPad

Knowledge articles display without styling when using the default CSS files on iOS 11.3. Articles are styled correctly on previous iOS versions. Analyzing the CSS file in the web inspector indicates that the CSS file is corrupt with additional whitespace, rendering it unusable.

The CSS produced is included below:

iOS 11.2:
 ```
html {
    -webkit-text-size-adjust: 100%;
    text-size-adjust: 100%;
}

img, iframe, table, pre {
    max-width: 100%;
    width: auto;
    height: auto;
}

pre {
    display: block;
    word-wrap: break-word;
    border: 1px solid #ccc;
    background: #efefef;
    padding: 8px;
}

header {
    font-size: 1.25em;
    font-weight: 800;
    font-style: normal;
    line-height: 1.5;
    margin-bottom: 1.25em;
    margin-top: 0.625em
}

field-title {
    font-size: 0.75em;
    font-weight: 800;
    font-style: normal;
    text-transform: uppercase;
    display: block;
    margin: 4pt 0pt;
}

field-value {
    font-size: 0.875em;
    font-weight: 300;
    font-style: normal;
    line-height: 143%;
    margin-bottom: 1.25em;
    max-width: 100%;
    overflow-wrap: break-word;
    word-wrap: break-word;
    word-break: break-word;
    -webkit-hyphens: auto;
    hyphens: auto;
}

body {
    background: rgba(255, 255, 255, 255);
    font-family: -apple-system, "Helvetica Neue";
}

body {
    margin: 0pt max(10pt, env(safe-area-inset-right)) env(safe-area-inset-bottom) max(10pt, env(safe-area-inset-left));
}

body, h1, h2, h3, h4, h5, h6 {
    color: rgba(0, 0, 0, 255);
}

a {
    color: rgba(98, 110, 173, 255);
}
```

iOS 11.3:
```
html  {
      - webkit - text - size - adjust: 100%;
     text - size - adjust: 100%;
     
}

 
img,  iframe,  table,  pre  {
     max - width:100%;
     width:auto;
     height:auto;
     
}


pre  {
     display:block;
     word - wrap:break - word;
     border:1px solid #ccc;
     background:#efefef;
     padding:8px;
     
}

 
header  {
     font - size:1.25em;
     font - weight:800;
     font - style:normal;
     line - height:1.5;
     margin - bottom:1.25em;
     margin - top:0.625em 
}

 
field - title  {
     font - size: 0.75em;
     font - weight:800;
     font - style:normal;
     text - transform: uppercase;
     display: block;
     margin: 4pt 0pt;
     
}

 
field - value  {
     font - size: 0.875em;
     font - weight:300;
     font - style:normal;
     line - height:143%;
     margin - bottom:1.25em;
     max - width:100%;
     overflow - wrap: break - word;
     word - wrap: break - word;
     word - break: break - word;
      - webkit - hyphens: auto;
     hyphens: auto;
     
}

 
body  {
     background: rgba(255, 255, 255, 255);
     font - family:  - apple - system,  "Helvetica Neue";
     
}


body  {
     margin: 0pt max(10pt,  env(safe - area - inset - right)) env(safe - area - inset - bottom) max(10pt,  env(safe - area - inset - left));
     
}


body,  h1,  h2,  h3,  h4,  h5,  h6  {
     color: rgba(0, 0, 0, 255);
     
}


a  {
     color: rgba(98, 110, 173, 255);
     
}
```

iOS 11.2:
![39377924-b75beb58-4a0b-11e8-94fd-741369ef9419](https://user-images.githubusercontent.com/1349921/39416163-a1ac16de-4bff-11e8-9a30-a76ad699f341.png)

iOS 11.3:
![39377925-b775302c-4a0b-11e8-80e6-d7b1c557563b](https://user-images.githubusercontent.com/1349921/39416166-a8f4c1c0-4bff-11e8-9286-ab33014e4d96.png)

**Comment by MikeNachbaur** [comment]
Hello @chrisperkins, thank you for this bug report.  I've been able to reproduce this locally, and have determined that this is caused by a regression in the behavior of WebKit in iOS 11.3.  I've filed a bug with Apple, and in the meantime have applied a fix that will work around this issue in our next release.

**Comment by chrisperkins** [comment]
Thanks @MikeNachbaur

**Comment by MikeNachbaur** [comment]
@chrisperkins We have released a patch to address this bug in our 212.0.3 release, so please give that version a try and let us know if you have any problems with it.

You can find the updated SDK on our releases page here: https://developer.salesforce.com/page/SnapinsMobile

**Comment by chrisperkins** [comment]
I updated to 212.0.3 and it works as advertised, thanks!
